server/src/main/java/io/seata/server/storage/redis/lock/RedisLocker.java

```diff
@@ -152,13 +153,15 @@ public boolean releaseLock(String xid, List<Long> branchIds) {
                             if (pipeline == null) {
                                 pipeline = jedis.pipelined();
                             }
```

Comment:
It can be deleted and the following with delKeys judgment

Reply:
ok